web/src/components/apps/EmbeddedClusterManagement.tsx

```diff
@@ -267,11 +275,40 @@ const EmbeddedClusterManagement = ({
         enableHiding: false,
         enableColumnDragging: false,
         size: 150,
+        Cell: ({ cell }) => {
+          const value = cell.getValue<string>();
+          return (
+            <Link
+              to={slug ? `/${slug}/cluster/${value}` : `/cluster/${value}`}
```

Comment:
Can you tell me more about these two URLs and what is different on each of those pages (what happens on that page when there's a slug in the URL vs not)?